recipes/restinio/config.yml

```diff
@@ -0,0 +1,3 @@
+versions:
+  "0.6.7.1":
+    folder: all
```

Comment:
Should wait for restinio/0.6.8 (https://github.com/conan-io/conan-center-index/pull/1635#issuecomment-628387328).